docs/sections/cli_features.md

```diff
@@ -262,6 +262,27 @@ popper ci travis \
 
 And the above will create a 3x3 matrix job for travis.
 
+## Translating workflows for other task runners
+
+The `translate` subcommand generates configuration files for other CI systems. The file generated by the `ci` subcommand executes Popper internally, the `translate` subcommand convert Popper workflows directly to the notation of the target CI system. The syntax of this command is the following:
+
+```bash
+popper translate --file wf.yml --to <service-name>
+```
+
+Where `<service-name` is the name of the CI system (see `popper translate --help` to get a list of supported systems).
```

Comment:
`<service-name` to `<service-name>`

Reply:
fixed de0cee23d492e2a302d19e8d296ba4e71001c7ee